eng/pipelines/checkout-task.yml

```diff
@@ -0,0 +1,11 @@
+# Checkout Roslyn sources on Windows
+steps:
+  - checkout: none
+
+  - bash: |
```

Comment:
According to docs, using this task doesn't install additional dependencies on Windows https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/bash?view=azure-devops